Error while checking build/O2/fullCI for c7d57115c6b99828ca7446a565258b188dcb0c98 at 2024-09-04 14:09:
```
## sw/BUILD/Clang-latest/log
  141 |   ExitOnError ExitOnErr(std::string(*argv[0]) + ": error:");
  201 |   ExitOnError ExitOnErr(std::string(*argv[0]) + ": error:");


## sw/BUILD/o2codechecker-latest/log
100% tests passed, 0 tests failed out of 1


## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/13478-slc8_x86-64/0/Detectors/GlobalTracking/src/MatchGlobalFwd.cxx:424:42: error: 'nFakes' was not declared in this scope
/sw/SOURCES/O2/13478-slc8_x86-64/0/Detectors/GlobalTracking/src/MatchGlobalFwd.cxx:424:53: error: 'nTrue' was not declared in this scope
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13478/c7d57115c6b99828ca7446a565258b188dcb0c98/build_O2_fullCI/pretty.html).

--- a/Detectors/GlobalTracking/src/MatchGlobalFwd.cxx
+++ b/Detectors/GlobalTracking/src/MatchGlobalFwd.cxx
@@ -408,6 +408,7 @@ void MatchGlobalFwd::doMatching()
       LOG(info) << "  MFT-MCH Matching: nFakes = " << nFakes << " nTrue = " << nTrue;
     }
   } else if constexpr (saveAllMode == SaveMode::kSaveNCandidates) {
+    int nFakes = 0, nTrue = 0;
     auto& matchAllChi2 = mMatchingFunctionMap["matchALL"];
     for (auto MCHId = 0; MCHId < mMCHWork.size(); MCHId++) {
       auto& thisMCHTrack = mMCHWork[MCHId];

